README.md

```diff
@@ -5,6 +5,8 @@
 
 [![npm version](https://badge.fury.io/js/intercom-client.svg)](http://badge.fury.io/js/intercom-client)
 
+### Does not currently support API V2.2
```

Comment:
It is my understanding that currently only the `Customer Search` binding is available from version 2.x of the API. The rest of bindings expect version 1.0. Would `Excluding Customer Search (2.0), only supports 1.0` sound better here? Let me know what you think.

Reply:
Yeah that is better. Thanks! made the update.